src/atcore.cpp

```diff
@@ -228,5 +228,15 @@ void AtCore::stop()
 
 void AtCore::pushCommand(const QString &comm)
 {
-    serial()->pushCommand(plugin()->translate(comm));
+    if (!plugin()) {
```

Comment:
this crashes.. instead use 
if (state() == DISCONNECTED || state() == CONNECTING)